app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine errors."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when visitor has no vaccine."""
+    pass

```

Comment:
Same here, for `NotVaccinatedError` class, you should provide a descriptive error message. For example:

```python
class NotVaccinatedError(VaccineError):
    def __init__(self):
        super().__init__('The person is not vaccinated.')
```